client/scripts/boinc-client.service.in

```diff
@@ -5,6 +5,18 @@ After=network-online.target
 
 [Service]
 ProtectHome=true
+PrivateTmp=true
+ProtectSystem=strict
+NoNewPrivileges=true
+ProtectKernelModules=true
+ProtectKernelTunables=true
+ProtectControlGroups=true
+RestrictRealtime=true
+RestrictAddressFamilies=AF_INET AF_INET6 AF_UNIX
+RestrictNamespaces=true
+PrivateUsers=true
+CapabilityBoundingSet=
+ReadWritePaths=/var/lib/boinc
```

Comment:
Needs to include `/etc/boinc-client`. Debian stores configuration files there and the client needs to be able to write to `cc_config.xml` and `global_prefs_override.xml`. (This means `gui_rpc_auth.cfg` and `remote_hosts.cfg` are protected only by filesystem permissions but that's the best we can do.)